src/App.js

```diff
@@ -337,17 +376,24 @@ function App() {
           </button>
         </div>
       </div>
-        {guessCount >= 5 && gameWon === false && (<div>
-          <button className="give-up-btn" onClick={giveUp}>
+        {guessCount >= 5 && gameWon === false && gaveUp === false && (<div>
+          <button className="give-up-btn" onClick={toggleShowGiveUpModal}>
             Give up?
           </button>
         </div>)}
+        {(gameWon === true || gaveUp === true) && (<div>
+          <button className="show-results-btn" onClick={() => setGameResultModal(true)}>
+            Show Results
+          </button>
+        </div>)}
       <div>
         <p className="game-timer">
           New round in {hours}:{minutes}:{seconds}
         </p>
       </div>
-      {/* <button onClick={resetGame}>reset</button>  */}
+      {process.env.NODE_ENV === 'development' && (
+        <button onClick={resetGame}>reset</button>
+      )}
```

Comment:
this way dev running `npm run dev` can show the reset button but production environment wont